tests/test_tftools.py

```diff
@@ -86,9 +86,9 @@ def test_batch_norm_2d():
     training = tf.placeholder(tf.bool, [])
     y = layers.batch_norm(x, training)
     sess = tf.Session()
-    sess.run(tf.initialize_all_variables())
-
     x_ = np.float32(np.random.randn(*input_shape))
+    
+    sess.run(tf.initialize_all_variables(), feed_dict={x:x_, training:True})
```

Comment:
https://www.tensorflow.org/api_docs/python/state_ops/exporting_and_importing_meta_graphs#initialize_all_variables

we should switch to `tf.global_variables_initializer`